packages/feedback/src/FeedbackForm.js

```diff
@@ -112,26 +113,29 @@ const FeedbackForm = ({
           feedbackApp: undefined,
           smileField: undefined,
         }}
-        validationSchema={yup.object().shape({
-          feedback: yup
-            .string()
-            .max(200, 'Additional Feedback cannot exceed 200 characters.')
-            .required('This field is required.'),
-          additionalFeedback: yup
-            .string()
-            .max(200, 'Additional Feedback cannot exceed 200 characters.'),
-          smileField: yup
-            .object()
-            .shape({
-              icon: yup.string().required(),
-              description: yup.string(),
-              placeholder: yup.string(),
-            })
-            .required('This field is required.'),
-          feedbackApp: yup
-            .string()
-            .isRequired(aboutOptions.length > 0, 'This field is required.'),
-        })}
+        validationSchema={
```

Comment:
On second thought, let’s add a new prop, ‘formProps’ that gets spread onto the ‘<Form />’, so that the developer can override any of the props for the ‘<Form />’, not just the ‘validationSchema’